### DISCUSSION TITLE
Questions about data privacy

### DISCUSSION THREAD

**Discussion by henrydrp10** [openai/whisper#1726] [Q&A]
Hello,

Sorry for the dumb question, but if I download whisper using pip install as it says in the readme, and I use it in my laptop, will any of the data be sent anywhere outside of my laptop? Or will it all stay in local, and the model will have been downloaded and executed locally too? 

Thank you!

**Comment by glangford** [comment]
All of your data is local, and the model is downloaded and executed locally using this repo of whisper. For more discussion, see

* https://github.com/openai/whisper/discussions/1424
* https://github.com/openai/whisper/discussions/1100
* https://github.com/openai/whisper/discussions/249
